spark/v3.2/build.gradle

```diff
@@ -82,6 +84,8 @@ project(':iceberg-spark:iceberg-spark-3.2') {
 
     testImplementation("org.apache.hadoop:hadoop-minicluster") {
       exclude group: 'org.apache.avro', module: 'avro'
+      // to make sure io.netty.buffer only comes from project(':iceberg-arrow')
+      exclude group: 'io.netty', module: 'netty-buffer'
```

Comment:
We don't actually need this line make jmh work with the new change in `iceberg-arrow`, but I still keep them to make sure netty buffer only comes from `iceberg-arrow`. I'm open to any suggestion.